hydra-api.yaml

```diff
@@ -258,6 +258,33 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+    delete:
+      summary: Deletes a project
+      parameters:
+        - name: id
+          in: path
+          description: project identifier
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: project deleted
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  redirect:
+                    type: string
+                    description: url of the Hydra instance
```

Comment:
Like, the root URL?

```suggestion
                    description: the root of the Hydra instance
```

Reply:
Yes, thanks, I always forget this wording :D